react/tic-tac-toe/src/app/screens/Login/components/customInput/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'src/scss/colors';
+@import 'src/scss/fonts';
+
+.inputs {
+  border-radius: 5px;
+  height: 32px;
+  margin: 8px 0;
+  width: 100%;
+}
+
+.inputValidate {
```

Comment:
normally we define clase like input-validate and use this in component like {styles.inputValidate}